flowzone.yml

```diff
@@ -3290,6 +3318,19 @@ jobs:
 
           echo "caps=${caps}" >> $GITHUB_OUTPUT
 
+      - name: Install cfn-lint
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install cfn-lint==0.83.7
+
+      - name: Lint template
```

Comment:
Is this just a linter for CF templates? I would prefer to review this in a separate PR from the environment stuff.

Reply:
in that case, we should do that one first as it prevents garbage from being committed.